draft-ietf-quic-transport.md

```diff
@@ -2205,12 +2210,13 @@ An endpoint is not expected to handle key updates when it is closing or
 draining.  A key update might prevent the endpoint from moving from the closing
 state to draining, but it otherwise has no impact.
 
-An endpoint could receive packets from a new source address, indicating a client
-connection migration ({{migration}}), while in the closing period. An endpoint
-in the closing state MUST strictly limit the number of packets it sends to this
-new address until the address is validated (see {{migrate-validate}}). A server
-in the closing state MAY instead choose to discard packets received from a new
-source address.
+While in the closing period, an endpoint could receive packets from a
+new source address, indicating a client connection migration
+({{migration}}). An endpoint in the closing state MUST strictly limit
+the number of packets it sends to this new address until the address
+is validated (see {{migrate-validate}}). A server in the closing state
+MAY instead choose to discard packets received from a new source
+address.
```

Comment:
The change to the fill-column is making this harder to review than I'd like.  I guess that's just a consequence of insisting on wrapping at <80...

Reply:
I tried rewrapping it for him, but that makes all the comments outdated, unfortunately.  I think this will need a rewrap just before merge instead, unfortunately.